ui/src/shared/components/TableGraph.js

```diff
@@ -29,26 +29,38 @@ class TableGraph extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const {data} = timeSeriesToTableGraph(nextProps.data)
+    const {data, unzippedData} = timeSeriesToTableGraph(nextProps.data)
 
     const {tableOptions: {sortBy: {internalName}}} = nextProps
     const sortByColumnIndex = _.indexOf(data[0], internalName)
 
     const sortedData = _.sortBy(_.drop(data, 1), sortByColumnIndex)
-    this.setState({data: [data[0], ...sortedData], sortByColumnIndex})
+    this.setState({
+      data: [data[0], ...sortedData],
+      unzippedData,
```

Comment:
does this need to be initialized in state?

Reply:
good catch- thank you !